test-engineering/load-tests/load_test_setup.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+#Setup MacOS - Install Cloud SDK
+echo -e "==================== Do you want to install GCP SDK on macOS? [yes or no]: "
+read response
+case $response in
+    [yY] | [yY][Ee][Ss] )
+        echo -e "==================== Install latest Cloud SDK version "
+        /usr/local/bin/wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-364.0.0-darwin-x86_64.tar.gz -P $DIRECTORY
+        tar -xf $DIRECTORY/google-cloud-sdk-364.0.0-darwin-x86_64.tar.gz
+        echo -e "==================== Do clean-up "
+        rm -rf $DIRECTORY/google-cloud-sdk-364.0.0-darwin-x86_64.tar.gz
+        echo -e "==================== Install SDK "
+        ./$SDK_DIRECTORY/install.sh
+        ;;
+    [nN] | [n|N][O|o] )
+        echo -e "==================== Skip GCP SDK Installation "
+        ;;
+    *) echo -e "==================== Invalid input it should be [yes or no] "
+        ;;
+esac
+
+#Setup environment
+GCLOUD=$(which gcloud)
+SED=$(which sed)
+KUBECTL=$(which kubectl)
+
+#RED_OUPUT='\033[0;31m'
+DIRECTORY=$(pwd)
+SDK_DIRECTORY=google-cloud-sdk
+MERINO_DIRECTORY=$DIRECTORY/kubernetes-config
+MASTER_FILE=locust-master-controller.yml
+WORKER_FILE=locust-worker-controller.yml
+SERVICE_FILE=locust-master-service.yml
+
+#Declare variables
+LOCUST_IMAGE_TAG=$(git log -1 --pretty=format:%h)
+#LOCUST_IMAGE_TAG=qwe123
+echo "Docker image tag for locust is set to: ${LOCUST_IMAGE_TAG}"
+
+##Kubernetes Manifests variables
+KINTO_SERVER_URL=firefox.settings.services.mozilla.com
+KINTO_COLLECTION=quicksuggest
+LOCUST_MASTER_NODE_HOST=locust-master
+KINTO_BUCKET=main
+CLUSTER=merino-load-test
+TARGET=stage.merino.nonprod.cloudops.mozgcp.net
+SCOPE="https://www.googleapis.com/auth/cloud-platform"
+REGION=us-central1
+ZONE=${REGION}-b
+PROJECT=spheric-keel-331521
+
+#Configure Kubernetes
+echo -e "==================== Prepare environments with set of environment variables "
+echo -e "==================== Set Kubernetes Cluster "
+export CLUSTER=$CLUSTER
+echo -e "==================== Set Kubernetes TARGET "
+export TARGET=$TARGET
+echo -e "==================== Set SCOPE "
+export SCOPE=$SCOPE
+
+echo -e "==================== Prepare GCP by updating default zone and project ID "
+$GCLOUD config set compute/zone $ZONE
+$GCLOUD config set project $PROJECT
+
+echo -e "==================== Refresh Kubeconfig at path ~/.kube/config "
+$GCLOUD container clusters get-credentials $CLUSTER --zone $ZONE --project $PROJECT
```

Comment:
Note: I had a permissions failure here, but the script kept on running (giving me other `builds submit` errors and `container images list`, etc errors in my logs. Not sure if we can get bash to fail hard after first error instead of continuing.

Reply:
Adding `set -eu` to the top of the file (usually the first line after the `#!` line), will cause bash to fail the script if any command fails or any variable is undefined. I'd highly recommend using it for all production scripts.

Reply:
```suggestion
$GCLOUD container clusters get-credentials $CLUSTER --zone $ZONE --project $GOOGLE_CLOUD_PROJECT
```